manim/utils/file_ops.py

```diff
@@ -56,4 +58,19 @@ def get_sorted_integer_files(directory,
         elif remove_non_integer_files:
             os.remove(full_path)
     indexed_files.sort(key=lambda p: p[0])
-    return list(map(lambda p: os.path.join(directory, p[1]), indexed_files))
\ No newline at end of file
+    return list(map(lambda p: os.path.join(directory, p[1]), indexed_files))
+
+def open_file(file_path):
+    current_os = platform.system()
+    if current_os == "Windows":
+        os.startfile(file_path)
+    else:
+        commands = []
+        if current_os == "Linux":
+            commands.append("xdg-open")
+        elif current_os.startswith("CYGWIN"):
```

Comment:
**Question :** Can `platform.system()` returns such a thing ? I thought it only returns four things (Java,Darwin, Windows, Linux). See https://docs.python.org/3/library/platform.html#platform.system

Will this work ?

Reply:
Yup. If the end user is using Python via a Cygwin installation, it will be something like `CYGWIN_NT=6.3`

https://stackoverflow.com/questions/32612235/what-is-os-name-on-cygwin